nextflow.config

```diff
@@ -31,6 +36,19 @@ params {
     ref_flat                     = "${params.dx_tracks_path}/rna/${params.gencode_version_name}.ref_annot.gtf.refflat"
     rrna_intervals               = "${params.dx_tracks_path}/rna/GRCh38_rRNA_genbank.interval_list"
 
+
+    // Reference sets for outrider
+
+    outrider_gene_chx_parents   = "/hpc/diaggen/projects/rnaseq_outrider_DxNextflowRNA1.0.1/sample_pools/chx/gene_set1_parents_20250702/"
+    outrider_gene_chx_index     = "/hpc/diaggen/projects/rnaseq_outrider_DxNextflowRNA1.0.1/sample_pools/chx/gene_set2_index_20250702/"
+    outrider_gene_cntrl_parents = "/hpc/diaggen/projects/rnaseq_outrider_DxNextflowRNA1.0.1/sample_pools/cntrl/gene_set1_parents_20250702/"
+    outrider_gene_cntrl_index   = "/hpc/diaggen/projects/rnaseq_outrider_DxNextflowRNA1.0.1/sample_pools/cntrl/gene_set2_index_20250702/"
+
+    outrider_exon_chx_parents   = "/hpc/diaggen/projects/rnaseq_outrider_DxNextflowRNA1.0.1/sample_pools/chx/exon_set1_parents_20250702/"
+    outrider_exon_chx_index     = "/hpc/diaggen/projects/rnaseq_outrider_DxNextflowRNA1.0.1/sample_pools/chx/exon_set2_index_20250702/"
+    outrider_exon_cntrl_parents = "/hpc/diaggen/projects/rnaseq_outrider_DxNextflowRNA1.0.1/sample_pools/cntrl/exon_set1_parents_20250702/"
+    outrider_exon_cntrl_index   = "/hpc/diaggen/projects/rnaseq_outrider_DxNextflowRNA1.0.1/sample_pools/cntrl/exon_set2_index_20250702/"
+
```

Comment:
Did you validate the workflow + config? 
Could you add the params to the nextflow_schema.json?
[link to parameters.md](https://github.com/UMCUGenetics/DxNextflowRNA/blob/feature/outrider/docs/parameters.md)
[link to nextflow_schema.json](https://github.com/UMCUGenetics/DxNextflowRNA/blob/feature/outrider/nextflow_schema.json)

[nf-core pipeline schema](https://nf-co.re/docs/nf-core-tools/pipelines/schema#/)